main.py

```diff
@@ -3,5 +1,53 @@
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
     pass
+    genres = ["Western",
+              "Action",
+              "Dramma"]
+
+    actors = ["George Klooney",
```

Comment:
There is a typo in the actor's name 'George Klooney'. It should be 'George Clooney'. This is corrected later in the code, but it's better to fix it here to avoid unnecessary updates.